### ISSUE TITLE
High loss/low accuracy when training an inception model on a new dataset

### ISSUE THREAD

**Issue by oakfr** [tensorflow/models#844]
Hi,

I am training an inception model on a new dataset of my own.

The dataset contains 643 classes, approx. 500K images for training and 50K images for validation (so approx 800 examples per class).

I am following the tutorial in models/inception to train an inception model from scratch (after having updated imagenet_data.py with the correct values).

The learning and the evaluation seem to be working properly, but I am surprised by the loss value during training (and the low final accuracy).

The training took approx 5 days before I stopped it.

The loss starts at around 13 and goes down smoothly to approx 5, but then does not go lower than that.  See graph below.

Here is my training command:

bazel-bin/inception/imagenet_train --num_gpus=8 --batch_size=64 --train_dir=/ops/tmp/large_train --data_dir=/ops/tf-data/large

Here are the last lines of training output:

2017-01-03 09:11:53.448907: step 416850, loss = 5.63 (43.2 examples/sec; 1.482 sec/batch)
2017-01-03 09:12:04.900361: step 416860, loss = 4.27 (89.7 examples/sec; 0.714 sec/batch)
2017-01-03 09:12:17.574107: step 416870, loss = 4.63 (53.2 examples/sec; 1.204 sec/batch)
2017-01-03 09:12:29.010834: step 416880, loss = 7.79 (51.9 examples/sec; 1.233 sec/batch)
2017-01-03 09:12:44.569015: step 416890, loss = 3.92 (42.4 examples/sec; 1.509 sec/batch)
2017-01-03 09:12:56.061142: step 416900, loss = 5.46 (58.9 examples/sec; 1.086 sec/batch)
2017-01-03 09:13:12.139927: step 416910, loss = 4.66 (43.9 examples/sec; 1.459 sec/batch)
2017-01-03 09:13:24.543821: step 416920, loss = 4.64 (56.1 examples/sec; 1.141 sec/batch)
2017-01-03 09:13:38.969471: step 416930, loss = 4.49 (48.8 examples/sec; 1.311 sec/batch)

Here is the output of the evaluation :

2017-01-03 09:27:48.529406: precision @ 1 = 0.3717 recall @ 5 = 0.5916 [50016 examples]

I was able to reach 80% accuracy on a simple dataset with 4 classes, so I believe my tensorflow / machine setup is fine.

I am running tensorflow 0.11.0 on a p2.8x.large AWS machine with CUDA 8.0 and cuDNN 5.1.5.

I did not change any hyper-parameter.

Any idea what my problem might be?  (beyond dataset quality)

Thanks!

![image](https://cloud.githubusercontent.com/assets/17097626/21604264/935db7e6-d1a2-11e6-93e8-c59ce7f092bc.png)

**Comment by michaelisard** [comment]
This type of question is better asked on [Stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow). Github issues are for bug reports and installation problems.
